lib/movable_ink/aws/ssm.rb

```diff
@@ -3,12 +3,12 @@
 module MovableInk
   class AWS
     module SSM
-      def ssm_client
-        @ssm_client ||= Aws::SSM::Client.new(region: 'us-east-1')
+      def ssm_client(region = nil)
+        @ssm_client ||= (region) ? region : Aws::SSM::Client.new(region: 'us-east-1')
```

Comment:
I might have missed it last time, but this will result in `region` variable assigned to `@ssm_client` if it provided as an argument for ssm_client.

```
@ssm_client ||= (region) ? region : Aws::SSM::Client.new(region: 'us-east-1')
```

is the short notation for the following:

```
if region
  @ssm_client ||= region
else
  @ssm_client ||= Aws::SSM::Client.new(region: 'us-east-1')
end
```

Reply:
fixed using default args,  version bumped, and rspec tests added and completed successfully

`➜  spec git:(ndoubleday/ssmregions) ✗ bundle exec rspec spec ssm_spec.rb
.......

Finished in 0.02632 seconds (files took 1.45 seconds to load)
7 examples, 0 failures`